src/graph/immutable_graph.cc

```diff
@@ -58,6 +134,8 @@ ImmutableGraph::EdgeArray ImmutableGraph::CSR::GetEdges(IdArray vids) const {
     dgl_id_t vid = vid_data[i];
     int64_t off = this->indptr[vid];
     const int64_t len = this->GetDegree(vid);
+    if (len == 0)
+      continue;
```

Comment:
Will this happen?

Reply:
yes, I ran into this corner case before.

Reply:
I mean if `len==0`, the code should not enter look in L133. Why this will be triggered?

Reply:
ah, `len` is reused. i should rename it.